src/custom-types/cleanbc/cleanbc-projects.php

```diff
@@ -0,0 +1,60 @@
+<?php
```

Comment:
this was not the intent of this theme to add cpt...

Reply:
They can be easily excluded but it leans into the FSE experience.

Reply:
for now, i'm ok with leaving this, in as we(you) are exploring options.